### ISSUE TITLE
config.h

### ISSUE THREAD

**Issue by Hadatko** [nghttp2/nghttp2#1202]
Hi,
i wanted ask about 
#ifdef HAVE_CONFIG_H
#  include <config.h>
#endif /* HAVE_CONFIG_H */

shouldn't be used 
#  include "config.h"
instead of
#  include <config.h>
?

And even better could be renamed config.h to something more specific like nghttp2_config.h.

**Comment by sarenameas** [comment]
I agree with this. Renaming this nghttp2_config.h helps with portability in an environment where there are possibly multiple config files.

**Comment by Hadatko** [comment]
I made this pull request: https://github.com/nghttp2/nghttp2/pull/1311

**Comment by Hadatko** [comment]
Looks like still an issue
